src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexUtilities.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+using Microsoft.Health.Fhir.Core.Features.Search;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Reindex
+{
+    public class ReindexUtilities
```

Comment:
There is another class called ReindexUtilities under `Shared.Core`. I am guessing one of them needs to be deleted?

Reply:
Indeed!  Removed it.